src/App.tsx

```diff
@@ -0,0 +1,72 @@
+import { useState, useEffect } from "react";
+import "./App.css";
+import { MusicLibrary } from "./components";
+import data from "./fake-data/albums.json";
+
+export interface RawData {
+  title: string;
+  artist: string;
+  artworkUrl: string;
+}
+
+export interface CleanData {
+  title: string;
+  artist: string;
+  artworkUrl: string;
+  imageAlt: string;
+}
+
+const App = () => {
+  const [albumsData, setAlbumsData] = useState<CleanData[]>([]);
+
+  const handleRemoveButtonClick = (index: number) => {
```

Comment:
As a suggestion for tightening things up further and avoiding passing these three functions down as props to components that don't use them directly, I'd define them in the Header.tsx file and opt instead just to pass the albumsData and setAlbumsData down the component tree.